storage-operator/pkg/controller/volume/slice.go

```diff
@@ -0,0 +1,43 @@
+package volume
```

Comment:
Commit message:

```diff
- Note that since Go standard library is doesn't provide basic operation
+ Note that since Go standard library doesn't provide basic operation
```

Reply:
Done